submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/parser/ExperimentSpecParser.java

```diff
@@ -182,18 +186,41 @@ private static V1PodTemplateSpec parseTemplateSpec(
 
       if (podSpec.getInitContainers() != null
           && podSpec.getInitContainers().size() > 0) {
-        String volumeName = podSpec.getInitContainers().get(0).getVolumeMounts()
-            .get(0).getName();
-        String path = podSpec.getInitContainers().get(0).getVolumeMounts()
-            .get(0).getMountPath();
-
-        V1VolumeMount mount = new V1VolumeMount();
-        mount.setName(volumeName);
-        mount.setMountPath(path);
 
+        List<V1VolumeMount> initContainerVolumeMounts =
+            podSpec.getInitContainers().get(0).getVolumeMounts();
```

Comment:
Why get the first item?

Reply:
There are 2 init containers. 1. for git sync process and 2. for environment setup. 0th init container has been created for git code sync process. As part of git sync, it is required to get some properties of corresponding init container and set the same in pod spec level as well. Hence, this step.